src/seedu/addressbook/commands/DeleteMultiCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.ArrayList;
+
+public class DeleteMultiCommand extends Command {
+
+    public static final String COMMAND_WORD = "deletemulti";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes multiple persons identified by the index number used in the last person listing.\n"
+            + "Parameters: INDEX\n"
```

Comment:
Ditto from above, not obvious that its multi-param by the wording

Reply:
INDEX [INDEX...]

Reply:
Same as above, thanks for the suggestion.